com.kaii.hgui.json

```diff
@@ -56,9 +56,8 @@
                 {
                     "type" : "git",
                     "url" : "https://github.com/kaii-lb/hgui-rust.git",
-                    "commit" : "713af543c6798ee8d96d7ed4d3655453ba7d5ed1"
+                    "commit" : "9665298ac56d2d612cbc48716f5beae7d2f5251a"
                 },
-                "generated-sources.json"
```

Comment:
I didn't say to remove this. I said the file was missing here.

Reply:
i didn't notice the "never", sorry for all the fuss, added the file and removed the vendor